pytensor/tensor/elemwise.py

```diff
@@ -181,6 +181,16 @@ def __init__(self, input_broadcastable, new_order):
         self.augment = sorted([i for i, x in enumerate(new_order) if x == "x"])
         self.drop = drop
 
+        input_ndim = len(input_broadcastable)
+
+        self.is_left_expand_dims = "x" in new_order and (
```

Comment:
Can rely on augment property I guess?
```suggestion
        self.augment and (
```